src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$main-size: 100px;
+$main-gap: 1px;
+$day-bg-color: #eee;
+$day-hover-bg-color: #ffbfcb;
+$day-names-in-week: (
+  'mon': 1,
+  'tue': 2,
+  'wed': 3,
+  'thu': 4,
+  'fri': 5,
+  'sat': 6,
+  'sun': 7,
+);
```

Comment:
Consider using a more descriptive and specific variable name for '$day-names-in-week'. The current name implies it's a list of names, while it actually maps short names to their respective day numbers. A name like '$weekday-short-to-number-map' might better convey the purpose of this map.